tsconfig.json

```diff
@@ -48,10 +48,7 @@
     //   "node_modules/@types",
     //   "packages/core/types"
     // ],
-    "types": [                                /* Type declaration files to be included in compilation. */
-      "node",
-      "jest"
-    ],
+    // "types": []                                /* Type declaration files to be included in compilation. */
```

Comment:
how come these are no longer required?

Reply:
I'm afraid I can't remember fully. I think added when I was working through the clashing of `@types/react` and `@types/react-native`